extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1417,11 +1424,13 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     }
 
     const closeGalleryAndShowTargetSlide = event => {
```

Comment:
nit: Using class methods for listener callbacks with more than a couple lines of logic can help keep things organized